inc/saiacl.h

```diff
@@ -2937,10 +2946,40 @@ typedef enum _sai_acl_entry_attr_t
      */
     SAI_ACL_ENTRY_ATTR_ACTION_SET_FORWARDING_CLASS = SAI_ACL_ENTRY_ATTR_ACTION_START + 0x34,
 
+    /**
+     * @brief Enable ARS monitoring for a destination that can be a LAG or nexthopgroup
+     *
+     * @type sai_acl_action_data_t sai_object_id_t
+     * @flags CREATE_AND_SET
+     * @objects SAI_OBJECT_TYPE_LAG, SAI_OBJECT_TYPE_NEXT_HOP_GROUP
+     * @default disabled
+     */
+    SAI_ACL_ENTRY_ATTR_ACTION_SET_ARS_MONITORING = SAI_ACL_ENTRY_ATTR_ACTION_START + 0x35,
+
+    /**
+     * @brief Enable ARS forwarding for a destination that can be a LAG or nexthopgroup
+     *
+     * @type sai_acl_action_data_t sai_object_id_t
+     * @flags CREATE_AND_SET
+     * @objects SAI_OBJECT_TYPE_LAG, SAI_OBJECT_TYPE_NEXT_HOP_GROUP
+     * @default disabled
+     */
+    SAI_ACL_ENTRY_ATTR_ACTION_ENABLE_ARS_FORWARDING = SAI_ACL_ENTRY_ATTR_ACTION_START + 0x36,
```

Comment:
can you please consider the addition of ARS oid to this action ?

Reply:
Yes, this makes sense. Added a new ACL action as set ARS object. This way now we can enable policy based ARS forwarding using ACL rules.

Reply:
So, you will remove SAI_ACL_ACTION_TYPE_ENABLE_ARS_FORWARDING and add a new ACL action using the ARS Oid as parameter ?